docs/extras/redis.md

```diff
@@ -0,0 +1,5 @@
+[Redis](https://redis.io/) is an in-memory caching system, much like [Memcached](memcached.md). While [Varnish](varnish.md) is generally used to improve performance for anonymous users, `redis` is used to improve the performance for logged in users.
+
+To enable Redis in Drupal VM, first make sure `redis` is in the list of `installed_extras` in your `config.yml`. Install the [Redis](https://www.drupal.org/project/redis) module and configure it as described in the [module's README](http://cgit.drupalcode.org/redis/tree/README.md).
```

Comment:
As always, you're awesome! I'll hold off on the merge, then—unless you want me to merge now, then we can just review the changes/cleanups in a follow-up PR. I'd like to tag a new release soon, since there are a few small bugs fixed.